topic2/src/Application.java

```diff
@@ -0,0 +1,8 @@
+public class Application {
+    public static void main(String[] args) {
+        System.out.println("Hello world!");
+    }
+
+    Persona persona = new Persona();
```

Comment:
propun sa apelezi nume, prenume si gen prin contructor acum ca sa nu le faci hardcode in clasa